lib/showoff_utils.rb

```diff
@@ -442,6 +442,10 @@ def self.setup(dir_name)
     when 'kramdown'
       Tilt.prefer Tilt::KramdownTemplate, "markdown"
 
+    when 'commonmarker'
+      require 'tilt/commonmarker'
```

Comment:
This should not be required once https://github.com/rtomayko/tilt/pull/282 is merged.